### PR TITLE
Improve sort by field documentation

### PR DESCRIPTION
**Pull request by sreekanth-cb**
Adding information about the "type" field while
requesting the sort by fields option.

### CODE REVIEW

**Comment by sreekanth-cb** [comment]
another one for review @amarantha-k.

File: modules/fts/pages/fts-sorting.adoc [hunk 1]
```diff
@@ -77,6 +77,8 @@ This query sorts search-results based on `reviews.ratings.Overall` — a field t
 When there are multiple values, the highest `Overall` ratings are used for sorting.
 Hotels with no `Overall` rating are placed at the end.
 
+It is highly recommended to specify the `type` of the sort fields like `string` for text fields, `date` for datetime fields or `number` for numeric fields while mentioning the sort fields in the sort section of the search request. This would help in fetching more accurate sort results.
```
Comment (amarantha-k) [inline modules/fts/pages/fts-sorting.adoc]: Suggested reword: 
To fetch more accurate sort results, we strongly recommended specifying the `type` of the sort fields in sort section of the search result. For example, `string` for text fields, `date` for datetime fields, or `number` for numeric fields.

Comment (amarantha-k) [reply modules/fts/pages/fts-sorting.adoc]: @sreekanth-cb Please let me know if the suggested reword looks ok.

Comment (sreekanth-cb) [reply modules/fts/pages/fts-sorting.adoc]: two minor fixes i guess..
"we strongly **recommend**" and "in sort section of the search **request**" ?

Comment (sreekanth-cb) [reply modules/fts/pages/fts-sorting.adoc]: Thanks @amarantha-k , Fixed it and added a few new lines to improve readability.. Let me know if that is an issue.

**Review by sreekanth-cb — COMMENTED** [review]
done!

**Review by amarantha-k — APPROVED** [review]
Thanks!

**Pull request merged**